app/views/stats/stats.php

```diff
@@ -7,43 +7,93 @@
     font-size: .7em;
   }
 </style>
+<script>
+$(function() {
+  $(".getStats").click(function (){
+    var method = this.id;
+    var button = $("#"+method);
+    $.getJSON("stats/"+method)
```

Comment:
This is an interesting function I haven't seen before. We should look more into it for use in other areas we do AJAX.